apps/extension/package.json

```diff
@@ -47,6 +47,7 @@
     "bignumber.js": "^9.1.1",
     "buffer": "^6.0.3",
     "dompurify": "^3.0.2",
+    "fp-ts": "^2.16.1",
```

Comment:
Extension previously did not specify this dependency, and was only building because it is specified in Namadillo, so added the same version here